libs/@hashintel/type-editor/src/entity-type-editor/property-list-card/property-type-form/expected-value-selector/shared/delete-expected-value-modal.tsx

```diff
@@ -67,6 +68,7 @@ export const DeleteExpectedValueModal = ({
         border: 1,
         borderColor: theme.palette.gray[20],
       })}
+      classes={{ root: fluidFontClassName }}
```

Comment:
For `Modal`, since it's a component in the design system, could we just add the class there? Or would that mess something up in HASH?

Reply:
Done